http-servers/src/main/java/com/palantir/remoting/http/server/JsonExceptionMapper.java

```diff
@@ -59,7 +60,9 @@ public final Response toResponse(T exception) {
                 error = SerializableError.of(message, exception.getClass(),
                         Arrays.asList(stackTrace));
             } else {
-                error = SerializableError.of(message, exception.getClass());
+                String errorId = UUID.randomUUID().toString();
+                log.error("error has been logged as {}", errorId, exception);
+                error = SerializableError.of(errorId, RuntimeException.class);
```

Comment:
^ yup. Although I think it may make more sense to default to `InternalServerError` since that doesn't leak much information but does indicate that there was a server side exception. Thoughts?